sky/backends/onprem_utils.py

```diff
@@ -132,7 +132,7 @@ def get_local_auth_config(cluster_name: str) -> List[str]:
     return config['auth']
 
 
-def get_admin_python(cluster_name: str) -> List[str]:
+def get_python_executable(cluster_name: str) -> str:
```

Comment:
nit: "admin python" still present in docstr?

Reply:
Good catch, thanks!